### PR TITLE
Hotfix/generator manager

### PR DESCRIPTION
**Pull request by SushiRoll53**
Generator was not generating variables, constructor nor dependencies.
First cause was the fact that generator had multiple managers, so whenever a variable, constructor or dependency were added to the manager, was getting lost. In other words, the manager helping writing the files was not the same manager receiving the fields.
This PR removed the unnecessary managers, helped plugins to be more easy to understand, & cleaned the code a bit.

### CODE REVIEW

File: lib/controllers/interpret.dart [hunk 1]
```diff
@@ -44,7 +44,7 @@ class Interpret {
     log = Logger(runtimeType.toString());
     this.projectName = projectName;
     _interpret._pbSymbolLinkerService = PBSymbolLinkerService();
-    _interpret._generationManager = PBFlutterGenerator(PBFlutterWriter());
+    // _interpret._generationManager = PBFlutterGenerator(PBFlutterWriter());
```
Comment (Ihuds577) [inline lib/controllers/interpret.dart]: Don't leave in commented code, if possible

File: lib/controllers/interpret.dart [hunk 2]
```diff
@@ -101,7 +101,7 @@ class Interpret {
     var currentContext = PBContext(
         jsonConfigurations:
             MainInfo().configurations ?? MainInfo().defaultConfigs);
-    currentContext.generationManager = _generationManager;
+    // currentContext.generationManager = _generationManager;
```
Comment (Ihuds577) [inline lib/controllers/interpret.dart]: Don't leave in commented code, if possible

File: lib/controllers/interpret.dart [hunk 3]
```diff
@@ -197,7 +197,7 @@ class Interpret {
...
```
Comment (Ihuds577) [inline lib/controllers/interpret.dart]: Don't leave in commented code, if possible

File: lib/generation/flutter_project_builder/flutter_project_builder.dart [hunk 4]
```diff
@@ -202,6 +203,8 @@ class FlutterProjectBuilder {
                   intermediateItem.node,
                   isSymbolsDir ? symbolFilePath : fileNamePath));
 
+          // intermediateItem.node.generator.manager = flutterGenerator;
```
Comment (Ihuds577) [inline lib/generation/flutter_project_builder/flutter_project_builder.dart]: Same as above.

File: lib/generation/generators/layouts/pb_scaffold_gen.dart [hunk 5]
```diff
@@ -7,9 +7,9 @@ import 'package:parabeac_core/interpret_and_optimize/entities/subclasses/pb_inte
 import '../pb_flutter_generator.dart';
 
 class PBScaffoldGenerator extends PBGenerator {
-  final PBGenerationManager manager;
+  // final PBGenerationManager manager;
```
Comment (Ihuds577) [inline lib/generation/generators/layouts/pb_scaffold_gen.dart]: Also same as above.

File: lib/interpret_and_optimize/entities/inherited_scaffold.dart [hunk 6]
```diff
@@ -122,7 +123,7 @@ class InheritedScaffold extends PBVisualIntermediateNode implements
   }
 
   // @override
-  // PBNakedPluginNode generatePluginNode(
```
Comment (Ihuds577) [inline lib/interpret_and_optimize/entities/inherited_scaffold.dart:125]: Why is this entire block commented out?

**Review by Ihuds577 — CHANGES_REQUESTED** [review]
See comments in code.

File: lib/interpret_and_optimize/entities/inherited_scaffold.dart (hunk 6 above)
Comment (SushiRoll53) [reply lib/interpret_and_optimize/entities/inherited_scaffold.dart:125]: Nothing was using it, so I remove it. 
PD: I am also cleaning the other comments

**Review by Ihuds577 — APPROVED** [review]
Looks good to me!

**Review by x64Eddie — APPROVED** [review]

**Pull request merged**